tooling/bundler/src/bundle/windows/msi/wix.rs

```diff
@@ -549,7 +549,9 @@ pub fn build_wix_app_installer(
   data.insert("shortcut_guid", to_json(shortcut_guid.as_str()));
 
   let app_exe_name = settings.main_binary_name().to_string();
-  data.insert("app_exe_name", to_json(app_exe_name));
+  // Note: `app_exe_name` kept around to not break custom wix templates depending on it
+  data.insert("app_exe_name", to_json(&app_exe_name));
+  data.insert("main_binary_name", to_json(app_exe_name));
```

Comment:
are you even using this one? doesn't look like it

Reply:
yeah it was leftover from previous attempt at [497b481](https://github.com/tauri-apps/tauri/pull/10952/commits/497b481f65d1d177c06b4116b8d7dea19b1fa93f), it is fine to keep for now